plugins/color_panel/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
I haven't reviewed the patch in general yet, but as an initial note: this file should not be checked in for any plugin.

Reply:
Deleted all of them and added a line to `.gitignore`. Thanks!